src/com/magento/idea/magento2plugin/actions/generation/data/dialog/EntityManagerContextData.java

```diff
@@ -0,0 +1,381 @@
+/*
```

Comment:
Let's call it Entity Creator. The  Entity manager, it's something different in Magento 2. It was just an initial idea. Let's rename it.

Reply:
This feature gonna be a game-changer. Great job!

Reply:
Thank you! I will do that in the next PR!